CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ nav_order: 1
 
 ## [X.Y.Z] - YYYY-MM-DD
 
+- Kafka Topic config options `unclean_leader_election_enable` deprecated
```

Comment:
could you please match the change log entry with this one: https://github.com/aiven/terraform-provider-aiven/blob/main/CHANGELOG.md#450---2023-06-14

Reply:
done